lib/pages/movie_details.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/widgets/movie/movie_presentation.dart';
+import 'package:movie_card/widgets/movie/overview.dart';
+import 'package:movie_card/widgets/movie/movie_details_custom_nav_bar.dart';
+import 'package:movie_card/widgets/movie/cast.dart';
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key, required this.title});
+  final String title;
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  static const double padFinal = 130;
+  static const String title = "The Super Mario Bros. Movie";
+  static const String date = "2023-04-05";
+  static const String overview = "While working underground to fix a water main, Brooklyn plumbers—and brothers—Mario and Luigi are transported down a mysterious pipe and wander into a magical new world. But when the brothers are separated, Mario embarks on an epic quest to find Luigi.";
+  static const double percentIndicator = 0.78;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Theme.of(context).colorScheme.onPrimaryContainer,
+      body: const SingleChildScrollView(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            MoviePresentation(title: title,date: date, posterName: "MarioBros_poster", backdropName: "MarioBros_backdrop", percent: percentIndicator),
+            Overview(overview: overview,),
+            Cast(),
+            SizedBox(height: padFinal,),  // A small gap so that the navbar does not cover the images
```

Comment:
Trailing comma in MoviePresentation, and please run the autoformat for this file (and the rest as well)